proto/brpb.proto

```diff
@@ -665,7 +673,7 @@ message DataFileGroup {
 }
 
 message DataFileInfo {
-    // SHA256 of the file.
+    // Logical checksum of all the kv pairs in the file.
```

Comment:
This isn't a logical checksum over raw kvs, the checksum was executed after compression if i recall right.

Reply:
discussed offline, this is pre-compression checksum of the file but not exactly logical too. It's the checksum of the plaintext file.